includes/common.php

```diff
@@ -986,3 +986,17 @@ function array_by_column($array, $column)
 {
     return array_combine(array_column($array, $column), $array);
 }
+
+/**
+ * Return the first word from a string.
+ *
+ * For example if passed an SNMP response that contains units such as
+ * "3.115 V", this function will simply return "3.115".
+ *
+ * @param  string  $value
+ * @return string (containing only the first word of $value)
+ */
+function first_word($value)
```

Comment:
\LibreNMS\Util\Number::cast()

Does what you want, can you please us that?

Reply:
Actually, the sensor code should already use Number::cast().  Are you sure this is needed?

Reply:
The MiB returns a string that ends with the units (in Volts) with a space between the number and the units.
I raised a case with PrimeKey and they declined to change this, as it's related to how they get the voltage from the HSM.

In any case let's see what happens if I don't call that function.

Reply:
```
$ php artisan tinker
>>> Number::cast('1.323 V')
=> 1.323
```

Reply:
Number::cast works